packages/analytics-adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@evolv-integrations/analytics-adapter",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
With the rename, these changes constitute a major version, _breaking_ bump. The GA, GTag, and AA integrations use this package with the `BaseAdapter` name;

Reply:
done